book_extractors/common/extractors/tests/test_base_extractor.py

```diff
@@ -70,6 +70,20 @@ def should_add_results_with_metadata_to_data_passed_in(self, extractor):
         assert metadata['mock']['errors'] == {}
 
 
+    class TestPassingParentResults:
+        def should_correctly_set_parent_results_in_extractor(self, extractor):
+            entry = {'text': 'test string entry'}
+            results_correct = {'extraction_results':
```

Comment:
Hmm, muistaakseni mainitsin ainakin suullisesti että se metadata objekti olisi hyvä saada myös passattua näiden resulttien lisäksi eteenpäin. Eli että objektin rakenne olisi jotain tyyliin
```
{
  `extraction_results`: {...},
  `extraction_metadata`: {...},
  `parent`: {...}
}
```
 Eli jos katsot ExtractionPipelinen process funkkaria, niin huomaat että siellä luodaan myös tyhjä metadata objekti. Mut joo toi metadata puuttu tuosta mun mockupista joten my bad.